packages/wonder-blocks-tokens/src/tokens/semantic-color.ts

```diff
@@ -1,18 +1,87 @@
 import {color} from "./color";
 
+const border = {
+    primary: color.fadedOffBlack16,
+    subtle: color.fadedOffBlack8,
+    strong: color.fadedOffBlack50,
+    inverse: color.white,
+};
+
 export const semanticColor = {
     /**
      * For buttons, links, and controls to communicate the presence and meaning
      * of interaction.
      */
     action: {
-        primary: {
-            default: color.blue,
-            active: color.activeBlue,
+        // For primary actions
+        progressive: {
+            default: {
+                border: "transparent",
+                background: color.blue,
+                foreground: color.white,
+            },
+            hover: {
+                border: color.blue,
+                background: color.blue,
+                foreground: color.white,
+            },
+            press: {
+                border: color.activeBlue,
+                background: color.activeBlue,
+                foreground: color.white,
+            },
```

Comment:
Should the `focus` state also have tokens associated with it?

Reply:
`focus` will have its own separate state, that is directly allocated in the `semanticColor.border.focus` token. This is something Caitlyn and I have discussed and the goal is that any focus on any component should use a blue outline, so that's the reason why I'm not including it here.

Reply:
Sounds good! There is also the disabled focus outline!